### DISCUSSION TITLE
Full Animation Nodes List of Node Types?

### DISCUSSION THREAD

**Discussion by blanchsb** [JacquesLucke/animation_nodes#1778] [Q&A]
Is there an easy to access list of all the animation nodes to be able to call them from a script?
Just a list is all. (i.e. the node Type)

**Comment by OmarEmaraDev** [comment]
```python
import animation_nodes
animation_nodes.base_types.AnimationNode.__subclasses__()
```

**Reply by blanchsb** [reply]
Oh beautiful. 
I am making a node creation node for Serpens and wanted to allow other devs to create nodes in this editor using a visual scripting node.
Thanks much.
